### PR TITLE
First try on Releases Page

### PR DESCRIPTION
**Pull request by hedgehog1833**
Should be too much, those lines come mostly from the package-lock.json. I did the release page for desktop with most of the filtering functionality.
What's still missing:
- Time filter
- Display announcement date when sorting by it
- Styling is not perfect
- Logged in user is expected

### CODE REVIEW

File: src/Components/Card/ReleaseCard.module.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+.releaseCardMedia {
+  height: to-rem(300);
+  width: to-rem(300);
+  object-fit: cover;
+}
+
+.cardContent {
+  display: flex;
+  flex-direction: column;
+}
```
Comment (DanielW1987) [inline src/Components/Card/ReleaseCard.module.scss:20]: In the other css files I use the `bem` notation and kebab-case class names. Would be good if we do this everywhere.

File: src/Components/Releases/ReleaseFilter.module.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@import '../../Styles/partials/01-settings/settings.colors';
+@import '../../Styles/partials/02-general/general.to-rem';
+
+.releaseFilter {
```
Comment (DanielW1987) [inline src/Components/Releases/ReleaseFilter.module.scss:4]: can you please kebab case here?

File: src/Components/Navbar/SearchBar/SearchBar.tsx [hunk 3]
```diff
@@ -17,7 +17,7 @@ const SearchBar = (props: SearchBarProps) => {
         autoComplete={'off'}
         fullWidth
         className={classes['search']}
-        placeholder={'Search Artist or Release...'}
+        placeholder={'Search Artist...'}
```
Comment (DanielW1987) [inline src/Components/Navbar/SearchBar/SearchBar.tsx:20]: I had actually intended to have a central search for artists and releases in the future.

File: src/Components/Releases/ReleaseList.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+export type ReleaseListProps = {
+  releases: Release[]
+  pagination: BackendPagination
+  handlePaginationChange: (event: React.ChangeEvent<unknown>, page: number) => void
+}
+
+const ReleaseList = (props: ReleaseListProps) => {
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column' }}>
```
Comment (DanielW1987) [inline src/Components/Releases/ReleaseList.tsx:16]: current convention is to not style inline via sx prop

File: src/Components/Releases/ReleaseList.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+            <ReleaseCard
+              artist={release.artist}
+              additionalArtists={release.additionalArtists}
+              albumTitle={release.albumTitle}
+              releaseDate={release.releaseDate}
+              announcementDate={release.announcementDate}
+              genre={release.genre}
+              type={release.type}
+              coverUrl={release.coverUrl}
+              reissue={release.reissue}
```
Comment (DanielW1987) [inline src/Components/Releases/ReleaseList.tsx:29]: why not passing the whole release into the card?

Comment (hedgehog1833) [reply src/Components/Releases/ReleaseList.tsx:29]: Yes 🤦‍♂️

File: src/Components/Navbar/SearchBar/SearchBar.tsx (hunk 3 above)
Comment (hedgehog1833) [reply src/Components/Navbar/SearchBar/SearchBar.tsx:20]: Yeah let's add it then :)

File: src/Components/Releases/ReleaseList.tsx (hunk 4 above)
Comment (hedgehog1833) [reply src/Components/Releases/ReleaseList.tsx]: Ah okay, I refactored all the usages 👍

**Review by DanielW1987 — APPROVED** [review]

**Pull request merged**